package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Infer strong typings for commander options and action handlers",
   "main": "index.js",
   "scripts": {
-    "test": "tsd",
-    "prepublish": "tsd"
+    "test": "tsd && jest",
+    "prepublish": "tsd && jest"
```

Comment:
Since the same command is executed, it can also be configured as follows.

```diff
+    "test": "tsd && jest",
+    "prepublish": "npm run test"
```

Reply:
Good suggestion, will do.